hacep/src/test/java/org/kie/hacep/UpdatableKieSessionTest.java

```diff
@@ -66,7 +66,8 @@ public void init() throws Exception {
 
     private void initSessionContextFromEmbeddKjar() {
         EnvConfig envConfig = EnvConfig.getDefaultEnvConfig();
-        kieContainer = KieContainerUtils.getKieContainer(envConfig, ks);
+        kieContainer = KieContainerUtils.getKieContainer(envConfig,
```

Comment:
Please format the code in the same way, half of the changes of this file are to split a statement in multiple lines and the other half to change them to single line :)

Reply:
fixed